src/sitemap.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset
+        xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9
+            http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+    <url>
+        <loc>http://bloomlibrary.org/</loc>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+        <loc>http://bloomlibrary.org/#/landing</loc>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+        <loc>http://bloomlibrary.org/#/support</loc>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+        <loc>http://bloomlibrary.org/#/installers</loc>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+        <loc>http://bloomlibrary.org/#/about</loc>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+        <loc>http://bloomlibrary.org/#/opensource</loc>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+        <loc>http://bloomlibrary.org/#/suggestions</loc>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+        <loc>http://bloomlibrary.org/#/browse</loc>
+        <changefreq>daily</changefreq>
+    </url>
```

Comment:
Another task, BL-2423, deals with those concerns. This task just involces helping Google find the factory password.